src/snowflake/cli/api/project/definition.py

```diff
@@ -50,6 +52,10 @@ def _get_merged_definitions(paths: List[Path]) -> Optional[Definition]:
     loader.add_constructor(
         yaml.resolver.BaseResolver.DEFAULT_MAPPING_TAG, _no_duplicates_constructor
     )
+    loader.add_constructor(
+        yaml.resolver.BaseResolver.DEFAULT_MAPPING_TAG, _no_duplicates_constructor
+    )
```

Comment:
is this duplicate line from the one before? Is it on purpose?

Reply:
Good catch! Fixed 👌